Sources/Bookmarks/BookmarkListViewModel.swift

```diff
@@ -74,7 +74,6 @@ public class BookmarkListViewModel: BookmarkListInteracting, ObservableObject {
             self.observer = nil
         }
     }
-
```

Comment:
Complete tangent, but I've never liked this rule, I feel like it robs us of an important organisational tool.

Reply:
I'm kinda with you there. Then again I feel like if I need to use multiple lines for organization, then `// MARK:` is the way.

Reply:
I just really like having an extra line in a bunch of contexts